server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -17,6 +17,7 @@ CobolVisitor.declarativeSameMsg=The following token cannot be on the same line a
 CobolVisitor.invalidDefMsg=Invalid definition for: %s
 CobolVisitor.misspelledWord=A misspelled word, maybe you want to put %s
 CobolVisitor.progIDIssueMsg=There is an issue with PROGRAM-ID paragraph
+CobolVisitor.paragraphNotDefined=The following paragraph is not defined: %s
```

Comment:
Look ok for me, but we should make sure with Scott what is the expected error message from the COBOL programmer point of view. It may be changed later on.